src/spec/payid-protocol.md

```diff
@@ -259,38 +246,41 @@ This specification defines the PayID protocol - an application-layer protocol, w
 ### Header Content-Length
   As defined in [RFC7230][], a request or response SHOULD include a `Content-Length` header when the message's length can be determined prior to being transferred. PayID protocol does not add any additional requirements over HTTP for writing Content-Length.
 
-### Header PayID-version (TODO)
-  The PayID client MUST include the PayID version request header field to specify the version of the PayID protocol used to generate the request.
+### Header PayID-Version
+  The PayID client MAY include the PayID version request header field to specify the version of the PayID protocol used to generate the request.
 
-  If present on a request, the PayID server MUST interpret the request according to the rules defined in the specified version of the PayID protocol or fail the request with an appropriate error response code.
+  If present on a request, the PayID server SHOULD interpret the request according to the rules defined in the specified version of the PayID protocol or fail the request with an appropriate error response code.
```

Comment:
`If present on a request, the PayID server SHOULD parse the PayID-Version header to interpret the request. If it does, PayID server MUST interpret the request according to the rules defined in the specified version of the PayID protocol or fail the request with an appropriate error response code.`